workflow/automation/tests/test_shared_workflow/test_install_shared.py

```diff
@@ -23,6 +23,7 @@ def test_install_simulation(set_up):
 
         # The pickle file was missing the 10th argument. This enables the vm to be checked.
         input_params.insert(10, True)
+        input_params.pop(9)  # Removes the vs_ref stat file parameter
```

Comment:
Lol

Reply:
Can we fix the pickle file? It's getting multiple band-aids.

Reply:
The test data is quite a mission to change at the moment. The current process involves logging into the test machine (I don't think I even have a working account on it), finding the right data, loading the data from pickle, editing it, saving back into pickle and running a test to make sure it works. Which is a lot of effort compared with adding a line or two to patch the test.
In the short term a few more careful patches won't hurt, however in the longer term I think testing needs a relatively large overhaul.

Reply:
@sungeunbae Thoughts?